docker-compose-full.yml

```diff
@@ -57,8 +57,7 @@ services:
       - 9998:3334
     volumes:
       - ../v3-website:/usr/src/app
-      - "./tmp/exercism-tooling-jobs:/tmp/exercism-tooling-jobs"
-      - "./tmp/exercism-tooling-jobs-efs:/tmp/exercism-tooling-jobs-efs"
+      - "./tmp/exercism:/tmp/exercism"
```

Comment:
I've simplified this to just map the `exercism` dir. I _could_ discern between the `exercism/efs` and `exercism/tooling-jobs` to have more fine-grained permissions, but I opted not to do this as the current format will allow us to rename things within the `exercism` directory freely and we won't break anything.